### PR TITLE
add translated text

### PR DESCRIPTION
**Pull request by belkhoujaons**
add translated text

### CODE REVIEW

**Comment by belkhoujaons** [comment]
please don't forget the hacktobor label

**Comment by devkiran** [comment]
@belkhoujaons Can you run `npm run format` and push the changes?

File: components/account/UpdateTheme.tsx [hunk 1]
```diff
@@ -2,17 +2,20 @@ import { ChevronUpDownIcon } from '@heroicons/react/24/outline';
 
 import { Card } from '@/components/shared';
 import useTheme from 'hooks/useTheme';
+import { useTranslation } from 'react-i18next';
```
Comment (devkiran) [inline components/account/UpdateTheme.tsx:5]:
```suggestion
import { useTranslation } from 'next-i18next';
```

**Review by devkiran — CHANGES_REQUESTED** [review]

File: components/account/UpdateTheme.tsx (hunk 1 above)
Comment (belkhoujaons) [reply components/account/UpdateTheme.tsx]: Done

**Review by devkiran — APPROVED** [review]

**Pull request merged**